src/pages/Main/Main.js

```diff
@@ -1,11 +1,110 @@
 import React from 'react';
+import styled from 'styled-components';
+import MainProduct from './MainProduct/MainProduct';
+import MainCarousel from './MainCarousel/MainCarousel';
+import MainShortCutItem from './MainShortCutItem/MainShortCutItem';
+
+const MAIN_SHORTCUT = [
+  { id: 1, img: 'shortcut_img_url', categoryName: '럭키박스' },
+  { id: 2, img: 'shortcut_img_url', categoryName: '남성추천' },
+  { id: 3, img: 'shortcut_img_url', categoryName: '여성추천' },
+  { id: 4, img: 'shortcut_img_url', categoryName: '셀럽픽' },
+  { id: 5, img: 'shortcut_img_url', categoryName: '이번주 브랜드관' },
+  { id: 6, img: 'shortcut_img_url', categoryName: '정가 아래' },
+  { id: 7, img: 'shortcut_img_url', categoryName: '인기 럭셔리' },
+  { id: 8, img: 'shortcut_img_url', categoryName: '연말 선물' },
+  { id: 9, img: 'shortcut_img_url', categoryName: '겨울 코디' },
+  { id: 10, img: 'shortcut_img_url', categoryName: '수수료 할인' },
+];
 
 const Main = () => {
   return (
-    <div>
-      <div>Main Page</div>
-    </div>
+    <MainWrapper>
+      <MainCarousel />
+      <MainBox>
+        <MainShortCutList>
+          {MAIN_SHORTCUT.map(({ id, categoryName }) => {
+            return <MainShortCutItem key={id} title={categoryName} />;
+          })}
```

Comment:
화살표 함수이고, 함수 body가 return문 밖에 없으니 body와 return문을 없애고 바로 return 해도 좋을 것 같습니다!
```suggestion
          {MAIN_SHORTCUT.map(({ id, categoryName }) => <MainShortCutItem key={id} title={categoryName} />)}
```